procrastinate/__init__.py

```diff
@@ -1,10 +1,12 @@
 from procrastinate import metadata as _metadata_module
 from procrastinate.aiopg_connector import PostgresConnector
 from procrastinate.app import App
+from procrastinate.job_context import JobContext
 from procrastinate.retry import BaseRetryStrategy, RetryStrategy
 
 __all__ = [
     "App",
+    "JobContext",
```

Comment:
Don't you think `JobDeferrer` should be added as well? Not saying it should be done with this PR though…

Reply:
When would it be needed in client code ?

Reply:
For type-checking. `JobDeferrer` is `configure_task`'s return type, and `configure_task` is part of the public API.